go/inst/instance_topology.go

```diff
@@ -2814,6 +2814,7 @@ func relocateReplicasInternal(replicas [](*Instance), instance, other *Instance)
 		return RepointTo(replicas, &other.Key)
 	}
 	// GTID
+	gtidErrorsMsg := ""
```

Comment:
Maybe we could initialize it to "none". This way the final result will be
`Additional Errors: none`
instead of
`Additional Errors: `
in case of not additional errors.

As the alternative (just a proposal) we could format the final string without `Additional Errors:` part if there are not additional errors.

Reply:
Indeed, good idea.  I'll revise